scene/gui/tab_container.cpp

```diff
@@ -31,368 +31,323 @@
 #include "message_queue.h"
 
 
-
 int TabContainer::_get_top_margin() const {
 
+	if (!tabs_visible)
+		return 0;
+
+	// Respect the minimum tab height.
 	Ref<StyleBox> tab_bg = get_stylebox("tab_bg");
 	Ref<StyleBox> tab_fg = get_stylebox("tab_fg");
-	Ref<Font> font = get_font("font");
 
-	int h = MAX( tab_bg->get_minimum_size().height,tab_fg->get_minimum_size().height);
+	int height = MAX(tab_bg->get_minimum_size().height,tab_fg->get_minimum_size().height);
```

Comment:
I originally renamed the whole function to `_get_tab_height`; but the other `tabs.cpp` has a `_get_top_margin` asweel, so I kept it. I can rename the variable tho.